packages/captp/CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [4.0.2](https://github.com/endojs/endo/compare/@endo/captp@4.0.1...@endo/captp@4.0.2) (2024-01-17)
+
+
+### Bug Fixes
+
+* explicitely harden prototypes ([3f98274](https://github.com/endojs/endo/commit/3f9827429dc79105230e8f5377dcc6a14038e9f5))
```

Comment:
typo

Reply:
I’m sure this is a typo extracted from the referenced commit message. I can try to patch these in a separate commit.

Reply:
(If we switch to `changesets`, the notes and version bump recommendation are mutable all the way to the moment of cutting a release. We can’t edit history.)